### PR TITLE
Initial commit of the GdocAnchorPreview extension.

### PR DESCRIPTION
**Pull request by mgilson**

### CODE REVIEW

File: GdocMediumAnchorPreview/demo/example.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<html>
+	<head>
```
Comment (zackmoy) [inline GdocMediumAnchorPreview/demo/example.html]: nit. as discussed, spaces over tabs.

File: GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ *
+ * var editor = new MediumEditor('.editable', {
+ *    extensions: {
+ *        anchorPreview: new GdocMediumAnchorPreview(),
+ *    },
+ *    targetBlank: true,
+ *    anchorPreview: {
+ *        hideDelay: 500
+ *    }
+});
```
Comment (zackmoy) [inline GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js]: nit. `* });` also missing another * on line 17

File: GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            var unlinkEl = el.querySelector('.medium-editor-toolbar-anchor-preview-inner-remove');
+            this.on(unlinkEl, 'click', this.handleUnlink.bind(this));
+
+            return el;
+        },
+
+        /** Unlink the currently active anchor. */
+        handleUnlink: function() {
+            var activeAnchor = this.activeAnchor;
+            if (activeAnchor) {
```
Comment (zackmoy) [inline GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js]: could this just be
```
if (this.activeAnchor) {
  ...
}
```

File: GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+var GdocMediumAnchorPreview;
+
+(function() {
+
+var AnchorPreview = MediumEditor.extensions.anchorPreview;
+GdocMediumAnchorPreview = MediumEditor.Extension.extend.call(
+    AnchorPreview, {
+
+        /** @override */
+        getTemplate: function () {
```
Comment (zackmoy) [inline GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js]: nit. as discussed, 2 spaces over 4 spaces.

File: GdocMediumAnchorPreview/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+Google Doc style link preview for [Medium Editor](https://github.com/yabwe/medium-editor)
+
+## Including
```
Comment (zackmoy) [inline GdocMediumAnchorPreview/README.md]: super nits coming: I've seen most repos use "Installation" vs. "Including."

File: GdocMediumAnchorPreview/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+To use the GdocAnchorPreview extension, you must include Medium Editor before
+including this extension.  This should be fairly straight-forward:
```
Comment (zackmoy) [inline GdocMediumAnchorPreview/README.md]: nit. remove "This should be fairly straight-forward:"

File: GdocMediumAnchorPreview/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+```html
+<script src="https://cdn.jsdelivr.net/medium-editor/latest/js/medium-editor.min.js"></script>
+<script src="/path/to/gdoc-medium-anchor-preview.js"></script>
+```
+
+## Minifying
```
Comment (zackmoy) [inline GdocMediumAnchorPreview/README.md]: no action required. My personal preference is to move the Minifying section after Config Options. I'd probably read through the options before deciding to minify.

File: GdocMediumAnchorPreview/README.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
+Google Doc style link preview for [Medium Editor](https://github.com/yabwe/medium-editor)
```
Comment (zackmoy) [inline GdocMediumAnchorPreview/README.md]: Can we add a screenshot so developers can preview the extension?

**Comment by zackmoy** [comment]
post-nits LGTM. Yay for OSS!

File: GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js (hunk 3 above)
Comment (mgilson) [inline GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js]: Yep.  At one point I was calling other methods in here that unset the activeAnchor, but then I realized I didn't need that...

File: GdocMediumAnchorPreview/README.md (hunk 7 above)
Comment (mgilson) [inline GdocMediumAnchorPreview/README.md]: Sure.  Done.

File: GdocMediumAnchorPreview/README.md (hunk 6 above)
Comment (mgilson) [inline GdocMediumAnchorPreview/README.md]: Done.

File: GdocMediumAnchorPreview/README.md (hunk 5 above)
Comment (mgilson) [inline GdocMediumAnchorPreview/README.md]: Ok.  Why not?

File: GdocMediumAnchorPreview/README.md (hunk 8 above)
Comment (mgilson) [inline GdocMediumAnchorPreview/README.md]: Good question.  We'd have to upload one and currently we don't have any CSS so it's not pretty.  (I'd rather wait until it is pretty).

File: GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js (hunk 2 above)
Comment (mgilson) [inline GdocMediumAnchorPreview/js/gdoc-medium-anchor-preview.js]: Oops.  Thanks.

File: GdocMediumAnchorPreview/README.md (hunk 8 above)
Comment (zackmoy) [inline GdocMediumAnchorPreview/README.md]: SGTM

**Pull request merged**